examples/integrations/azure_devops_server.json

```
@@ -0,0 +1,14 @@
+{
+    "name": "Azure DevOps Server Sync",
+    "configType": "AZURE_DEVOPS_SERVER",
+    "url": "https://azure-devops.onprem.swaggerhub.com",
```

Consider using an `example.com` domain instead. Also, I think default installations have `/tfs` in their base path / API root.
```suggestion
    "url": "https://azure-devops.example.com/tfs",
```